sphinx6-preparations.rst

```diff
@@ -0,0 +1,37 @@
+.. post:: Dec 5, 2022
```

Comment:
I think we can publish this after the newsletter, and just mention we're working on it.

Reply:
It's pushed now, I think we should get the theme pre-release out before publishing this.

Reply:
See: https://github.com/readthedocs/sphinx_rtd_theme/pull/1397